src/caret_analyze/plot/visualize_lib/bokeh/stacked_bar.py

```diff
@@ -223,7 +154,7 @@ def _get_x_width_list(x_values: list[float]) -> list[float]:
         return x_width_list
 
     @staticmethod
-    def _add_shift_value(
+    def add_shift_value(
```

Comment:
Move this function to the StackedBarSource class.

Reply:
Fixed it in [7413d0f](https://github.com/tier4/CARET_analyze/pull/307/commits/7413d0f72a3ce8e0052c712da4b7db406081e5be).